pkg/api/v1alpha1/cluster_types.go

```diff
@@ -870,6 +870,33 @@ const (
 	MachineInvalidReason FailureReasonType = "MachineInvalid"
 )
 
+// ClusterCNIStatus is a type for defining statuses for Cluster default cni.
+type ClusterCNIStatus string
```

Comment:
```suggestion
type ClusterCNIState string
```

nit: to avoid confusion between status objects